ethportal-peertest/src/jsonrpc.rs

```diff
@@ -67,13 +80,25 @@ fn validate_endpoint_response(method: &str, result: &Value) {
                 ENR_SEQ.to_string()
             );
         }
+        "portal_stateFindNodes" => {
+            assert_eq!(
+                result.get("total").unwrap().as_u64().unwrap(),
+                peertest_nodes.nodes.len() as u64
+            );
+            assert!(result
+                .get("enrs")
+                .unwrap()
+                .as_str()
+                .unwrap()
+                .contains("enr:"));
+        }
```

Comment:
A better assertion would be to use the `FromStr` implementation of `Enr` to validate each ENR in the response.